js/plugins/google-genai/src/googleai/gemini.ts

```diff
@@ -621,6 +621,7 @@ export function defineModel(
             ...getBasicUsageStats(request.messages, candidateData),
             inputTokens: response.usageMetadata?.promptTokenCount,
             outputTokens: response.usageMetadata?.candidatesTokenCount,
+            reasoningTokens: response.usageMetadata?.thoughtsTokenCount,
```

Comment:
This needs to match GenerationUsageSchema
so it needs to be  thoughtsTokens: z.number().optional(),